### PR TITLE
Cleanup and standardize DevTools release notes

### PR DESCRIPTION
**Pull request by parlough**
This PR cleans up DevTools release note entries: expanding introductions, better following our writing and formatting guidelines, providing some extra information, and fixing a few spelling, grammar, punctuation, and other mistakes.

**Staged:** https://flutter-docs-prod--pr9325-feature-devtools-rel-46yfqcky.web.app/tools/devtools/release-notes

### CODE REVIEW

**Comment by parlough** [comment]
This can still be reviewed, but I'm going to hold off on landing it as I'll need to first land some changes in the DevTools repository.

File: src/tools/devtools/release-notes/index.md
Comment (atsansone) [inline src/tools/devtools/release-notes/index.md:14]: **issue:** This reads awkwardly. Consider something that mirrors the [Golang release notes](https://go.dev/doc/devel/release).

```suggestion
This page summarizes the changes between official stable releases of the DevTools.
To view a complete list of changes, check out the
[DevTools git log](https://github.com/flutter/devtools/commits/master).

The Dart and Flutter SDKs include the DevTools.
To check your version of DevTools, run the following on your command line:

```terminal
dart devtools --version
```
```

File: src/tools/devtools/release-notes/index.md
Comment (atsansone) [inline src/tools/devtools/release-notes/index.md:40]: **issue:** Move this farther up with the other explanation.

```suggestion

```

File: src/tools/devtools/release-notes/release-notes-2.10.0-src.md
Comment (atsansone) [inline src/tools/devtools/release-notes/release-notes-2.10.0-src.md:7]: **question:** Does DevTools pull this release note as a single Markdown file or a rendered HTML page? According to [this include](https://github.com/flutter/website/pull/9325/files#diff-34fa2c96c654719ae827c942cb1ea0533d21242f931522b114661b0bf4f05d20R7), it must not.

I ask because this would be better served as an include with a variable to ensure consistency.

File: src/tools/devtools/release-notes/release-notes-2.21.1-src.md
Comment (atsansone) [inline src/tools/devtools/release-notes/release-notes-2.21.1-src.md:69]: **question:** Again with the use of an include. That may work better here, too.

**Review by atsansone — CHANGES_REQUESTED** [review]
@parlough : There appears to be some conflict as to whether or not this supports includes. If it does, consider refactoring for includes with variables. That would work better throughout.

File: src/tools/devtools/release-notes/release-notes-2.10.0-src.md
Comment (parlough) [reply src/tools/devtools/release-notes/release-notes-2.10.0-src.md:7]: At least currently we can't use constructors like the Jekyll variables or Liquid includes. I'll consider ways to do so in the future.

File: src/tools/devtools/release-notes/release-notes-2.21.1-src.md
Comment (parlough) [reply src/tools/devtools/release-notes/release-notes-2.21.1-src.md:69]: At least currently we can't use constructors like the Jekyll variables or Liquid includes. I'll consider ways to do so in the future.

**Comment by parlough** [comment]
We currently can't use site variables and various Liquid functionality. Perhaps with some other changes in the future.

**Review by sfshaza2 — APPROVED** [review]
lgtm! But I won't land, as per your request.

**Comment by parlough** [comment]
@atsansone Please take another look when you get a chance :)

**Review by atsansone — APPROVED** [review]
Check your verb tense on the entries. I approve pending that resolution.

File: src/tools/devtools/release-notes/release-notes-2.10.0-src.md
Comment (parlough) [reply src/tools/devtools/release-notes/release-notes-2.10.0-src.md:19]: I'll leave that for separate work since I didn't touch these. A decision will have to made with the DevTools team to standardize on either present or past tense for their changelog entries.

**Pull request merged**
